services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -3,7 +3,7 @@ info:
   title: simcore-service-webserver
   description: ' Main service with an interface (http-API & websockets) to the web
     front-end'
-  version: 0.31.0
+  version: 0.32.0
 servers:
```

Comment:
Q: Where do you change this version?

Reply:
@matusdrobuliak66 use `make version-minor` in the corresponding service or package.